core/server/api/canary/members.js

```diff
@@ -184,7 +184,7 @@ module.exports = {
         permissions: true,
         async query(frame) {
             try {
-                frame.options.withRelated = ['stripeSubscriptions'];
+                frame.options.withRelated = ['stripeSubscriptions', 'labels'];
```

Comment:
Now that we don't automatically overwrite labels (and thus include them in the result) - we have to explicitly request labels in the result